src/Makefile

```diff
@@ -5,11 +5,12 @@ all: shvec.out
 shvec.out: shvec.c shvec.h
 	gcc $(FLAGS) shvec.c -o shvec.out
 
-test: shvec.c shvec.h
-	echo "Tests are running!"
+test: test_shvalloc.out
+	echo "Tests are starting..."
+	./test_shvalloc.out
 
 clean:
 	rm shvec.out
 
-test_shvalloc.out: minunit.h test_shvalloc.c
-	gcc -g -Wall test_shvalloc.c -o test_shvalloc.out
\ No newline at end of file
+test_shvalloc.out: minunit.h test_shvalloc.c shvec.c shvec.h
```

Comment:
wouldn't we have a seperate test for shvec?
I was thinking of _test_shvec.c_
_test_shvalloc_ only includes tests for the custom malloc (shvalloc)

Reply:
oh I see. I'll fix this